api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -132,6 +134,14 @@ def convert_id_into_label(value_id: str | None) -> str | None:
     return convert_id_into_label
 
 
+class gtls(BaseModel):
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** Case :smile: classe -> PascalCase ‏
```suggestion
class GtlLabels(BaseModel):